src/Microsoft.AspNetCore.Hosting.Abstractions/IHostingEnvironment.cs

```diff
@@ -17,6 +17,13 @@ public interface IHostingEnvironment
         /// </summary>
         // This must be settable!
         string EnvironmentName { get; set; }
+        
+        /// <summary>
+        /// Gets or sets the name of the application. This property is automatically set by the host to the assembly containing
+        /// the application entry point.
+        /// </summary>
+        // This must be settable!
```

Comment:
I simply copied this comment as per the other properties. I don't think it affects compilation though